### PR TITLE
Update api service example with new paths/api

### LINKED ISSUE #7: examples/api/services out of date
https://github.com/ipfs/examples/tree/master/examples/api/service

The example doesn't build anymore, mostly because packages moved around but possibly some of the API also changed.
- `code.google.com/p/go.net/context` should be `golang.org/x/net/context`
- `github.com/jbenet/go-ipfs/...` should be `github.com/ipfs/go-ipfs/...`
- `~/.go-ipfs` should be `~/.ipfs`

### PR DESCRIPTION
**Pull request by shazow**
Fixes #7 so that the examples build.

Edit: Error resolved.

### CODE REVIEW

**Comment by shazow** [comment]
Also, would be swell to get similar examples into the go-ipfs codebase so that it shows up in godoc. Happy to do a PR if you're +1.

**Comment by shazow** [comment]
Getting this:

```
machine1 $ go run host.go
I am peer: QmbEHFxrkEhioRuT6hLJjFPZuPCuXVJEKdUGAHBweywTdJ

machine2 $ go run client.go QmbEHFxrkEhioRuT6hLJjFPZuPCuXVJEKdUGAHBweywTdJ
I am peer <peer.ID a8Ux6N> dialing <peer.ID 9tmz13>
failed to find any peer in table
```

Any ideas?

File: examples/api/service/host.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	fmt.Printf("I am peer: %s\n", nd.Identity)
+	fmt.Printf("I am peer: %s\n", nd.Identity.Pretty())
```
Comment (shazow) [inline examples/api/service/host.go:35]: Is this the right format we want to output? Seems there are multiple encoding formats available.

**Comment by jbenet** [comment]
Hey @shazow o/

> Also, would be swell to get similar examples into the go-ipfs codebase so that it shows up in godoc. Happy to do a PR if you're +1.

:+1: 

> failed to find any peer in table

i think this is because our bootstrap peers have been supper sluggish lately. We're investigating + will fix soon. this certainly wasn't happening a week ago.

**Pull request merged**